zou/app/services/persons_service.py

```diff
@@ -334,66 +339,43 @@ def invite_person(person_id):
     Send an invitation email to given person (a mail telling him/her how to
     connect on Kitsu).
     """
+    person = get_person(person_id)
     organisation = get_organisation()
-    person = get_person_raw(person_id)
+    token = str(uuid.uuid4())
```

Comment:
The token length may be too small for brute force attacks. It's recommended to use at least 64 chars.

Reply:
It's done!